consumer.go

```diff
@@ -2,11 +2,14 @@ package cluster
 
 import (
 	"sort"
+	"regexp"
 	"time"
 
 	"gopkg.in/Shopify/sarama.v1"
 )
 
+var validChars *regexp.Regexp = regexp.MustCompile(`\A[A-Za-z0-9._-]*\z`)
```

Comment:
Could you please rename this to `validGroupIDChars`? Are you sure consumer group names can start with non alphanumeric characters?